app/views/admin/consultations/_consultation_ministry.html.slim

```diff
@@ -0,0 +1,19 @@
+- if selected_ministry
+	- unless selected_ministry.is_approved
+		.row
+			.col-md-6
+				= link_to "Awaiting moderation", admin_ministry_path(selected_ministry.id), class: "float-right", 'data-toggle': "tooltip", 'data-placement': "right", title: "Approve ministry"
+		.admin-update-button-position
+			div class="btn disabled" data-toggle="tooltip" data-placement="bottom" title="Approve ministry" Submit Consultation
+	- else
+		.admin-update-button-position
+			button class="btn" type="submit" Submit Consultation
+- elsif (consultation&.ministry&.is_approved || consultation&.ministry_id.blank?)
```

Comment:
@BALAJI-BALA - not sure if the `&` is required here. Consultation must belong to a ministry.